gems/bess/spec/lib/houdini/nonprofit_creation_spec.rb

```diff
@@ -17,90 +24,110 @@
+		# 	let(:options) { { confirm_admin: false } }
+		# 	it { is_expected.to be_confirmed }
+		# 	it { is_expected.to_not be_super_admin }
+		# end
 	end
 
-	describe 'super_admin option' do
+	describe 'super_admin_option' do 
+		subject{ super(); User.find_by(email: user_email) }
 		let(:options) { { super_admin: true } }
-
-		it 'creates a user with super_admin role' do
-			subject
-			expect(User.find_by(email: user_email).super_admin?).to be_truthy
-		end
+		it { byebug; is_expected.to be_super_admin }
+		it { byebug; is_expected.to_not be_confirmed }
 	end
 
-	describe 'skip admin confirmation' do
-		let(:options) { { confirm_admin: false } }
+	# it 'returns successful hash' do
+	# 	creation_result = nonprofit_creation_result
+	# 	nonprofit_id = Nonprofit.find_by(name: nonprofit_name).id.to_s
+	# 	expect(creation_result).to eq({ success: true, messages: ["Nonprofit #{nonprofit_id} successfully created."] })
+	# end
 
-		it 'creates a confirmed user' do
-			subject
-			expect(User.where(email: user_email).last.confirmed?).to be_truthy
-		end
-	end
+	# describe 'super_admin option' do
+	# 	let(:options) { { super_admin: true } }
+	# 	subject {}
+		
+	# 	it 'creates a user with super_admin role' do
+	# 		nonprofit_creation_result
+			
+	# 	end
+	# end
 
-	describe 'nonprofit can not be saved' do
-	let(:result) {
-			{
-				nonprofit: {
-					name: nonprofit_name,
-					state_code: nonprofit_state_code,
-					city: nonprofit_city,
-					website: nonprofit_website,
-					email: nonprofit_email,
-					phone: nonprofit_phone
-				}, 
-				user: {
-					name: user_name,
-					email: nil,
-					password: 'P@ssw0rd!'
-				}
-			}
-		}
+	# describe 'skip admin confirmation' do
+	# 	let(:options) { { confirm_admin: false } }
+	# 	us
+	# 	it {is_expected.to }
+	# 	it 'creates a confirmed user' do
+	# 		nonprofit_creation_result
+	# 		expect(User.where(email: user_email).last).to be_confirmed
+	# 	end
+	# end
 
-		it 'returns failed hash with error messages' do
-			creation_result = subject
-			expected_error_messages = ["Error creating user: Email can't be blank", "Error creating user: Email is invalid"]
-			expect(creation_result).to eq({ success: false, messages: expected_error_messages })
-		end
-	end
+	# describe 'nonprofit can not be saved' do
+	# 	let(:result) do
+	# 		{
+	# 			nonprofit: {
+	# 				name: nonprofit_name,
+	# 				state_code: nonprofit_state_code,
+	# 				city: nonprofit_city,
+	# 				website: nonprofit_website,
+	# 				email: nonprofit_email,
+	# 				phone: nonprofit_phone
+	# 			},
+	# 			user: {
+	# 				name: user_name,
+	# 				email: nil,
+	# 				password: 'P@ssw0rd!'
+	# 			}
+	# 		}
+	# 	end
+
+	# 	it 'returns failed hash with error messages' do
+	# 		creation_result = nonprofit_creation_result
+	# 		expected_error_messages = ["Error creating user: Email can't be blank", 'Error creating user: Email is invalid']
+	# 		expect(creation_result).to eq({ success: false, messages: expected_error_messages })
+	# 	end
+	# end
```

Comment:
Do you think it is unnecessary to test error conditions?

Reply:
Oops, that was an accident. They should be tested.

Reply:
Addressed